includes/class-sensei-question.php

```diff
@@ -835,6 +930,8 @@ public static function answer_feedback_notes( $question_id ) {
 	 * Pseudo code for logic:  https://github.com/Automattic/sensei/issues/1422#issuecomment-214494263
 	 *
 	 * @since 1.9.0
+	 *
+	 * @deprecated 3.14 Moved into the_answer_feedback
 	 */
 	public static function the_answer_result_indication() {
```

Comment:
Probably we should also call the `_deprecated_function` here.

Reply:
Yup adding them in f5198bc